Freshli.Web/Views/Home/Index.cshtml

```diff
@@ -10,17 +10,21 @@
 
 <div>
     <form asp-route="CreateAnalysisRequest" method="post">
-        <div>
-            <label asp-for="Url">Public Git URL:</label>
-            <input asp-for="Url" type="url"/>
+        <div class="form-group">
+            <label asp-for="Name">Name</label>
+            <input asp-for="Name" class="form-control" />
         </div>
-        <div>
-            <label asp-for="Name">Your name:</label>
-            <input asp-for="Name"/>
+        <div class="form-group required">
+            <div>
+                <label asp-for="Email">Email Address</label>
+                <input asp-for="Email" type="email" class="form-control" required/>
+            </div>
         </div>
-        <div>
-            <label asp-for="Email">Your email address:</label>
-            <input asp-for="Email" type="email"/>
+        <div class="form-group required">
+            <div>
+                <label asp-for="Url">Public Git URL</label>
+                <input asp-for="Url" type="url" class="form-control" placeholder="https://github.com/corgibytes/freshli" required/>
```

Comment:
I don't think Freshli isn't the best placeholder, because we don't have support for .NET projects, yet.

Reply:
I updated this to be `https://github.com/legacycoderocks/website`, but I don't think that's actually the current website repository...

The metrics are quite bad... 📈